apps/api-extractor/src/api/ExtractorConfig.ts

```diff
@@ -135,6 +135,13 @@ export interface IExtractorConfigPrepareOptions {
    * `@microsoft/api-extractor/extends/tsdoc-base.json`.
    */
   tsdocConfigFile?: TSDocConfigFile;
+
+  /**
+   * When preparing the configuration object, folder and file paths referenced in the configuration are checked
+   * for existence, and an error is reported if they are not found.  This option can be used to disable this
+   * check. This may be useful when preparing a configuration file for an un-built project.
+   */
+  ignoreMissingConfigTargets?: boolean;
```

Comment:
This only ignores missing declarations, right? This name implies that all referenced files (including the `extends` target) are ignored if they're missing.